src/page/ConfigPageLayout/tabs/UninitializedTab.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { EmptyState, TextLink } from '@grafana/ui';
+
+import { ROUTES } from 'types';
+import { AppInitializer } from 'page/AppInitializer';
+
+import { ConfigContent } from '../ConfigContent';
+
+export function UninitializedTab() {
+  // For some reason the 'call-to-action' variant causes infinity loop in the test (if the image is shown)
+  const hideImage = process.env.NODE_ENV === 'test';
```

Comment:
Madness 🤕 I think this is okay to leave even though adding test specific code to production components feels smelly.

Reply:
It would be `const hideImage = true` in production, but I agree with the smellyness either way.